forecasting/auto_ts.py

```diff
@@ -0,0 +1,199 @@
+from forecasting.model_selection import cross_val
+from forecasting.more_models import *
+from forecasting.graphs import resid_diagnostic, future_forecast,cross_val_graph
+from forecasting.normal_naive_models import naive_pi, drift_pi, mean_pi
+import pandas as pd
+import plotly.graph_objects as go
+from sklearn.metrics import *
+
+
+
+
+class Forecast:
+
+    def __init__(self, data:pd.DataFrame,target_col:str=None, period:int=1):
+        
+        """
+        :param data: pandas.DataFrame - Historical time series data with a date-time index.
+        :param target_col: str - Column with historical data. If none will default to the first column.
+        :param period: int - Seasonal period.
+        """
+
+        self.data = data
+        self.period = period
+
+        if not target_col:
+            target_col = self.data.columns[0]
+
+        self.target_col = target_col
+
+    def select_best_model(self, n_splits:int=5, test_size:int=None, models:dict=None, time_taken:bool=False):
```

Comment:
Add minimisation error score as argument (string input). Add available strings in docstring.

Always return time_taken inside dataframe (remove bool option)